### PR TITLE
build: add @angular/bazel dependency for Bazel

### PR DESCRIPTION
**Pull request by brandonroberts**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,6 +119,7 @@
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.901.5",
     "@angular-devkit/build-ng-packagr": "~0.901.5",
+    "@angular/bazel": "^9.1.5",
```
Comment (alex-okrushko) [inline package.json:122]: That's the one that is being deprecated, right?
https://github.com/angular/angular/pull/37190

Why do we need it? 🙂

Comment (brandonroberts) [reply package.json:122]: It's needed in the current toolchain we use for the `ng_module`, and `ng_package` rules. 
https://github.com/ngrx/platform/blob/master/tools/defaults.bzl#L5

**Comment by alex-okrushko** [comment]
```sh
 /home/circleci/project/node_modules/.bin/tsc -p modules/schematics/tsconfig.schematics.json
(node:221) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
```
That's the `build-affected` failure, right?

**Comment by brandonroberts** [comment]
That's not the failure. When building all the packages, there is an issue of build order I'm looking into, but will be a separate PR.

**Pull request merged**